content/installation/python/PythonSupportedTechnologies.md

```diff
@@ -1,46 +1,70 @@
-<!-- 
+<!--
 title: "Supported Technologies"
 description: "List of supported technologies"
 tags: "installation Python agent frameworks support package"
 -->
 
-The Python agent supports Python versions 2.7 and 3.5 to 3.8. Framework support is currently for:
+## Contrast agents
 
-* Django: 1.10+ and 2.0+ <br> (Django 2 is available for Python 3 only.)
-* Flask: 0.10 - 0.12 and 1.0+
-* Pyramid: 1.9
-* Pylons: 1.0+
+Contrast agents are supported for one year after release. Older agents may continue to function and remain compatible, but they are no longer fully supported.
 
->**Note:**
-* The Python agent is meant to be WSGI compatible. It may be compatible to other WSGI applications as long as the guidelines are followed.
+Contrast applies bug fixes and develops new features on the latest version of the agent. Code changes are not backported to previous versions. While a workaround may be provided for a bug, to resolve issues, you should update to the release in which the issue was addressed.
 
+## Operating systems
 
-## Database Support
+Agent testing is done on **64-bit OSX** and **64-bit Linux**.
 
-The Python agent has database support for:
+Starting with version 2.3.0 of the agent, the package installation step requires the compilation of C extensions. This process is automatic, but it requires that certain software is installed in the target environment:
 
-* MySQL (`PyMySQL and mysql-connector`)
-* SQLite3 (`sqlite3` and `pysqlite2`)
-* PostgreSQL (`psycopg2`)
+* At a minimum, `gcc`, `make`, `automake` and `autoconf` are required. The package names may be different on different platforms. It may be useful to install your platform's version of `build-essential`.
+* On some platforms, it may be necessary to install system headers.
 
-## NoSQL Support
+## Python versions
 
-The Python agent has NoSQL support for:
+Contrast supports Python Long-Term Support (LTS) versions in **bugfix** and **security** status as well as the `2.7` version. Contrast shifts its support for Python versions as the working group shifts its LTS windows. See the [Status of Python branches](https://devguide.python.org/#status-of-python-branches) for specific release dates.
 
-* Mongo (`pymongo`)
 
-## Database Support
+### Supported
 
-The Python agent has ORM support for:
+* 3.8.X: First supported agent was 2.8.0
+* 3.7.X: First supported agent was 1.5.0
+* 3.6.X: First supported agent was 1.0.0
+* 3.5.X: First supported agent was 1.0.0
+* 2.7.X: First supported agent was 1.0.0
+
+### Deprecated
+
+* 3.4.X: Last supported agent was 1.9.0
+
+## Web servers
+
+* [gunicorn](https://gunicorn.org/) 20.0.X
```

Comment:
Since other sections add a preceding paragraph, maybe adding "Many Python frameworks can be run with various web servers. Currently the python agent can be run with ..."

Reply:
Adding a new row for "Web servers" in the new documentation with this content.